presqt/api_v1/helpers/read_write_tools.py

```diff
@@ -0,0 +1,42 @@
+import json
```

Comment:
Couple of things to think about as the number of "helpers" expands.

1. Would it be good to rename helpers -> utilities/utils?
2. Can we further sub-package utilities/helpers according to their purposes?
  * Ex. `io` subpackage, which then has write_file and zip_file inside of it.
3. It might be good to be consistent about only defining one "thing" per module, whether that it a top-level function or a class. It can feel like overkill at this stage, but keeps the code nice and clean as it grows.
4. Perhaps import all the helper/utilities into the __init__.py file for the package so that it can be imported more easily elsewhere.